homeassistant/components/tankerkoenig/manifest.json

```
@@ -1,9 +1,10 @@
 {
   "domain": "tankerkoenig",
   "name": "Tankerkoenig",
+  "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/tankerkoenig",
   "requirements": ["pytankerkoenig==0.0.6"],
```

Since we (you) are modifying it anyway, you might want to use the latest version of the library. As far as I could see it's only license information changes.
```suggestion
  "requirements": ["pytankerkoenig==0.0.7"],
```

---

dependency bumps should always go in a separate PR

---

btw. the latest version does not contain any functional changes, so bump is not necessary
further, not sure why 0.0.7 is the [last published](https://pypi.org/project/pytankerkoenig/#history) one, even a tag for [0.0.8](https://github.com/ultrara1n/pytankerkoenig/releases/tag/0.0.8) existent 🤨